src/game/Tactical/Interface_Panels.cc

```diff
@@ -947,6 +950,31 @@ void InitializeSMPanel()
 	InitInvSlotInterface(g_ui.m_invSlotPositionTac, &g_ui.m_invCamoRegion, SMInvMoveCallback, smInvClickCallback, SMInvMoveCamoCallback, SMInvClickCamoCallback);
 	InitKeyRingInterface(KeyRingItemPanelButtonCallback);
 
+	if (gamepolicy(informative_popups)) {
+		//Define regions for the informative popups displaying effective attribute values
+		MSYS_DefineRegion(&gSM_SELMERCStatsRegion[ATTR_AGILITY], dx + SM_AGI_X - 10, dy + SM_AGI_Y, dx + SM_AGI_X + SM_STATS_WIDTH, dy + SM_AGI_Y + 10, MSYS_PRIORITY_HIGH, MSYS_NO_CURSOR, SelectedMercPopupMoveCallback, MSYS_NO_CALLBACK);
+		MSYS_SetRegionUserData(&gSM_SELMERCStatsRegion[ATTR_AGILITY], 0, ATTR_AGILITY);
+		MSYS_DefineRegion(&gSM_SELMERCStatsRegion[ATTR_DEXTERITY], dx + SM_DEX_X - 10, dy + SM_DEX_Y, dx + SM_DEX_X + SM_STATS_WIDTH, dy + SM_DEX_Y + 10, MSYS_PRIORITY_HIGH, MSYS_NO_CURSOR, SelectedMercPopupMoveCallback, MSYS_NO_CALLBACK);
+		MSYS_SetRegionUserData(&gSM_SELMERCStatsRegion[ATTR_DEXTERITY], 0, ATTR_DEXTERITY);
```

Comment:
If the userdata of region[x] is always x itself you don't really need it because you can just compute it inside the callback handler: `auto attr = static_cast<Attributes>(pRegion - gSM_SELMERCStatsRegion)`;